bundle-size/common.js

```diff
@@ -49,6 +49,66 @@ exports.getCheckFromDatabase = async (db, headSha) => {
  * @param {number} delta the bundle size delta in KB.
  * @return {string} formatted bundle size delta.
  */
-exports.formatBundleSizeDelta = delta => {
+const formatBundleSizeDelta = delta => {
   return 'Δ ' + (delta >= 0 ? '+' : '') + delta.toFixed(2) + 'KB';
 };
+
+/**
+ * @param {string} file
+ * @param {string} description
+ * @return {string}
+ */
+const formatFileItem = (file, description) => `* \`${file}\`: ${description}`;
+
+exports.formatFileItem = formatFileItem;
+
+/**
+ * @param {{file: string, bundleSizeDelta: number}} item
+ * @return {string}
+ */
+exports.formatBundleSizeItem = ({file, bundleSizeDelta}) => {
+  return formatFileItem(file, formatBundleSizeDelta(bundleSizeDelta));
+};
+
+/**
+ * @param {string} file
+ * @return {string}
+ */
+const noExtension = file => {
+  const parts = file.split('.');
+  parts.pop();
+  return parts.join('.');
+};
```

Comment:
This is basically "the opposite of `path.extname`", so how about use it in the definition of this function and make it a oneliner:
```suggestion
const noExtension = file => file.slice(0, -path.extname(file).length);
```
(Add `const path = require('path');` at the top, ofc)

Tested this locally to make sure it works, e.g.:
```
> fn = 'a/b/adasds.max.js';
'a/b/adasds.max.js'
> const noExtension = file => file.slice(0, -path.extname(file).length);
undefined
> noExtension(fn);
'a/b/adasds.max'
```